src/plugins/gc-subway/_base.scss

```
@@ -16,20 +16,69 @@
 		border-top: 4px solid #26374a;
 		margin-top: 38px;
 
-		ul {
+		ul, dl {
 			clear: both;
 			list-style: none;
 			padding-left: .57em;
 			padding-top: 10px;
 			position: relative;
 
+			dt {
+				border-left: 4px solid #26374a;
+				font-weight: normal;
+				padding: 0px 20px 3px 1em;
+
+				:first-child::before {
+					background-color: #fff;
+					border: 3px solid #26374a;
+					border-radius: 50%;
+					content: "";
+					height: 1.3em;
+					left: 0em;
+					position: absolute;
+					transition: width .2s, height .2s, left .2s, margin-top .2s;
+					width: 1.3em;
+				}
+
+				&.active> :first-child::before {
```

Add a space before "<" character

```suggestion
				&.active > :first-child::before {
```